.rhcicd/clowdapp-backend.yaml

```diff
@@ -132,6 +132,69 @@ objects:
           value: https://3ff0dbd8017a4750a1d92055a1685263@o271843.ingest.sentry.io/5440905?environment=${ENV_NAME}
         - name: QUARKUS_LOG_SENTRY_ENVIRONMENT
           value: ${ENV_NAME}
+- apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: notifications-db-cleaner-config
+  data:
+    clean.sh: |
+      cat /notifications-db-cleaner/clean.sql | psql > /dev/null
+    clean.sql: |
+      CALL cleanEventLog();
+      CALL cleanKafkaMessagesIds();
```

Comment:
Wondering, if it fails, do we get an alert?

Reply:
Good question!

The answer depends on the failure:
- It could fail because of an OpenShift template change, but we should see the failure after the new template has been deployed.
- It could fail because of a SQL schema change that would break the stored procedures and then unfortunately the pod cronjob pod will not be marked as failed. I'll try to add a unit test that will execute the stored procedures during the notifications build. That should be enough to prevent a schema issue.
- It could probably fail for other reasons...

Reply:
The stored procedures are now tested with Java. This protects us against a breaking SQL schema change.

Reply:
> Is there any way for the openshift jobs to alert us that it's failing?

I don't know yet but I will look into that.